src/panels/config/devices/ha-config-devices-dashboard.ts

```diff
@@ -286,6 +312,116 @@ export class HaConfigDeviceDashboard extends LitElement {
   }
 
   protected render(): TemplateResult {
+    const activeFilters = this._activeFilters(
+      this.entries,
+      this._searchParms,
+      this.hass.localize
+    );
+
+    const headerToolbar = html`
+      <search-input
+        no-label-float
```

Comment:
Copied from `/src/panels/config/entities/ha-config-entities.ts`, could be moved to a separate component to avoid duplicating the code.